src/Network/Receive/kRO/RagexeRE_2020_03_04a.pm

```diff
@@ -0,0 +1,61 @@
+#########################################################################
+#  OpenKore - Packet Receiveing
+#  This module contains functions for Receiveing packets to the server.
+#
+#  This software is open source, licensed under the GNU General Public
+#  License, version 2.
+#  Basically, this means that you're allowed to modify and distribute
+#  this software. However, if you distribute modified versions, you MUST
+#  also distribute the source code.
+#  See http://www.gnu.org/licenses/gpl.html for the full license.
+########################################################################
+# Korea (kRO) #bysctnightcore
+# The majority of private servers use eAthena, this is a clone of kRO
+package Network::Receive::kRO::RagexeRE_2020_03_04a;
+use strict;
+use base qw(Network::Receive::kRO::Ragexe_2018_11_14c);
```

Comment:
better to use this:
```perl
use base qw(Network::Receive::kRO::Ragexe_2018_11_14c);```

Reply:
it same ?

Reply:
```perl
use base qw(Network::Receive::kRO::RagexeRE_2018_11_21);
```

Reply:
~~use base qw(Network::Receive::kRO::Ragexe_2018_11_14c);
use base qw(Network::Receive::kRO::RagexeRE_2018_11_21);

Reply:
@ya4ept  Ragexe_2018_11_14c for Ragexe not RagexeRE

Reply:
you write RagexeRE_** which is based on Ragexe _** ?